scripts/scrub-user.js

```diff
@@ -29,32 +35,27 @@ const parseResults = (results, email) => {
   const lastDatasetDate = isValidDate(process.env.LAST_DATASET_DATE)
     ? new Date(process.env.LAST_DATASET_DATE)
     : new Date();
-  const clipTotal = results.reduce((acc, curr) => acc + curr.clip_count, 0);
-  const clientIds = results.map(each => each.client_id);
 
-  const summary = {
+  return {
     email,
     length: results.length,
-    ids: clientIds,
-    clipTotal: clipTotal,
+    ids: results.map(each => each.client_id),
+    clipTotal: results.reduce((acc, curr) => acc + curr.clip_count, 0),
     firstClip: results[0] ? results[0].first_clip : null,
     inPastDataset: results[0] && lastDatasetDate >= results[0].first_clip,
   };
-
-  return summary;
 };
 
-const readlineLoop = (prompt, options) => {
-  readline.question(prompt, answer => {
-    const callback = options[answer.toLowerCase()];
-    if (callback) callback();
-    else readlineLoop(prompt, options);
-  });
+const readlineLoop = async (prompt, options) => {
+  const answer = await promptAsync(prompt);
+  const callback = options[answer.toLowerCase()];
+
+  if (callback) await callback();
+  else await readlineLoop(prompt, options);
```

Comment:
Not important, but I wonder if this would be cleaner if we returned the promise instead:

```
// untested, might break something 🙃
return callback ? callback() : readlineLoop(prompt, options);
```